index.bs

```diff
@@ -2502,7 +2502,7 @@ partial dictionary MLOpSupportLimits {
   </tr>
   <tr>
     <td>*output*</td>
-    <td>{{type}}</td>
+    <td>[=/any data type|any=]</td>
```

Comment:
Note: Same here, allowed data types, not the specified data type

Reply:
"Any type" is not allowed though for the output tensor, as the only *allowed* data type is the one that exactly matches `type`. So if we're reframing the description of these tables from their tensor constraints to literal `opSupportLimits` values, then let's search and replace "Constraints for op" with "opSupportLimits values for op".

Reply:
Done with https://github.com/webmachinelearning/webnn/pull/857/commits/ad2f04b50f522c24b5fa9a020a2f6787a754072c, PTAL.